config/settings.py

```diff
@@ -34,7 +34,9 @@
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
+    "django.contrib.sites",
```

Comment:
what can we reduce from this?

Reply:
nothing, they're needed because of the use of `tahoe-sites`

Reply:
understood 👍🏼 , and Done